### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by MieszkoGulinski**
new branch product_page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .env
 .nyc_output
 coverage
+.DS_Store
```
Comment (krzkaczor) [inline .gitignore]: please remove from local `.gitignore` and add it to you global ignore file.

File: page/partials/product_menu.ejs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div id="product-menu">
```
Comment (krzkaczor) [inline page/partials/product_menu.ejs]: is this file used anywhere?

File: page/sass/common.scss [hunk 3]
```diff
@@ -213,3 +216,26 @@ p.slim {
...
+        margin-top: 1px;
+        border-bottom: 5px solid $blue-darker;
+        width: 36px;
+    }
+    h5 {
+        margin-bottom: 5px;
+        font-weight: $font-weight-light;
+    }
+}
+.text-box.text-box-purple {
```
Comment (krzkaczor) [inline page/sass/common.scss]: move within the previous block as `&.text-box-purple`

File: page/sass/product.scss [hunk 4]
```diff
@@ -0,0 +1,255 @@
+[data-entrance] {visibility: hidden;} // required for JS animation to work
```
Comment (krzkaczor) [inline page/sass/product.scss]: could you wrap everything in .product-page class? (in html as well) so id doesnt leak to other sites

File: page/partials/product_menu.ejs (hunk 2 above)
Comment (MieszkoGulinski) [reply page/partials/product_menu.ejs]: Yes, it's the right side menu. `ul` element is automatically filled by JS, for every section that has in productContent non-empty string with menu text.

**Pull request merged**